code/entities/machine/soda-fountain/interactable/Dispenser.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Sandbox;
+using Sandbox.util;
+
+namespace Cinema;
+
+public partial class Dispenser : BaseInteractable
+{
+    static private float LeverPosIncrement = 0.05f;
+    static private float DispenseTime = 3.5f;
+    private string AnimationName { get; set; }
+    private TimeUntil DispenseTimer = 0.0f;
```

Comment:
`TimeUntil` should have a prefix and explain what it does, rename to `TimeUntilFinishedDispensing`

Reply:
Changed DispenseTimer to TimeUntilFinishedDispensing